src/bundle/Security/Http/Authentication/DefaultAuthenticationRequiredHandler.php

```diff
@@ -46,7 +55,7 @@ public function onAuthenticationRequired(Request $request, TokenInterface $token
     {
         // Do not save the target path when the current one is the one for checking the authentication code. Then it's
         // another redirect which happens in multi-factor scenarios.
-        if (!$this->isCheckAuthCodeRequest($request) && $request->hasSession() && $request->isMethodSafe(false) && !$request->isXmlHttpRequest()) {
+        if (!$this->isCheckAuthCodeRequest($request) && $request->hasSession() && $request->isMethodSafe() && !$request->isXmlHttpRequest()) {
```

Comment:
Missed this when cleaning up for 4.4, will take it over as a clean-up change.

Reply:
Added to https://github.com/scheb/2fa/commit/dd1d48194488b009ff482674021350b33d1487c6